README.md

```diff
@@ -33,6 +33,7 @@ If you have time and willing to help, there are a lof of ways to contribute:
 | [consul](https://github.com/netdata/go.d.plugin/tree/master/modules/consul)               | `Consul`                   |
 | [coredns](https://github.com/netdata/go.d.plugin/tree/master/modules/coredns)             | `CoreDNS`                  |
 | [couchdb](https://github.com/netdata/go.d.plugin/tree/master/modules/couchdb)             | `CouchDB`                  |
+| [dnsdist](https://github.com/netdata/go.d.plugin/tree/master/modules/dnsdist)             | `Dnsdist`                  |
```

Comment:
```suggestion
| [dnsdist](https://github.com/netdata/go.d.plugin/tree/master/modules/dnsdist)             | `DNSdist`                  |
```

I am not sure, but it seems the app is [`DNSdist`](https://www.open-xchange.com/products/dnsdist/)